### ISSUE TITLE
[Monitoring V2] Add default dashboard on Grafana Homepage

### ISSUE THREAD

**Issue by aiyengar2** [rancher/rancher#29308]
**What kind of request is this (question/bug/enhancement/feature request):**
feature request

The default Grafana page should show some Rancher Monitoring specific dashboards. We can configure this via the `grafana.ini` in the following way on chart deployment (where home.json is the key on the ConfigMap that stores the dashboard):

```yaml
grafana:
  grafana.ini:
    dashboards:
      default_home_dashboard_path: /tmp/dashboards/home.json 
```

![Bildschirmfoto 2020-10-01 um 17 28 08](https://user-images.githubusercontent.com/13127443/94849064-b25a9700-03d9-11eb-851e-3894ae39d2a1.png)

Thanks @bashofmann for coming up with the dashboard!

**Comment by jiaqiluo** [comment]
this is validated in master-head cb9e8a1 with the lastest chart 

Open the Grafana page from the Monitoring Overview page, the home page shows the following default dashboard 

![screencapture-167-172-120-123-k8s-clusters-c-76jd2-api-v1-namespaces-cattle-monitoring-system-services-http-rancher-monitoring-grafana-80-proxy-2020-10-01-17_47_06](https://user-images.githubusercontent.com/6218999/94877081-637b2480-040e-11eb-9721-08c638dbdbdc.png)

**Comment by aiyengar2** [comment]
@jiaqiluo we need to also validate this dashboard for logged in users. You can follow the same steps as https://github.com/rancher/rancher/issues/29308#issuecomment-702469881 but then navigate to the login page, log in with the default credentials (`admin/prom-operator`) and make sure that we don't encounter the error noted in https://github.com/rancher/charts/pull/756#issue-497089970

**Comment by jiaqiluo** [comment]
The bug fix is validated in rancher:master-head 87e7cdf

When we log in as the default user

![screencapture-167-172-192-82-k8s-clusters-c-hqx9m-api-v1-namespaces-cattle-monitoring-system-services-http-rancher-monitoring-grafana-80-proxy-2020-10-02-14_42_45](https://user-images.githubusercontent.com/6218999/94972315-d8596780-04bd-11eb-9027-add9f4027bc2.png)

Then signed out and reopen the Grafana link from the monitoring overview page 

![screencapture-167-172-192-82-k8s-clusters-c-hqx9m-api-v1-namespaces-cattle-monitoring-system-services-http-rancher-monitoring-grafana-80-proxy-2020-10-02-14_48_41](https://user-images.githubusercontent.com/6218999/94972567-67667f80-04be-11eb-8168-e20a9e4d957a.png)
